index.bs

```diff
@@ -339,13 +340,60 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: A user agent implementing, in conjunction with the underlying platform, the [=Web Authentication API=] and algorithms
     given in this specification, and handling communication between [=authenticators=] and [=[RPS]=].
 
+: <dfn>Credential ID</dfn>
+:: A probabilistically-unique [=byte sequence=] identifying a [=public key credential source=].
+
+    Credential IDs are generated by [=authenticators=] in two forms:
+    1. Bytes with at least 100 bits of entropy, or
+    1. The [=public key credential source=], without its [=public key credential source/id=] item, encrypted so only the owning
+        authenticator can decrypt it.
+
+    [=[RPS]=] don't need to distinguish these two kinds of Credential IDs.
+
```

Comment:
All the text above also already exists in the spec, and so can be removed from this PR.